src/gc/env/gcenv.os.h

```diff
@@ -128,6 +128,9 @@ typedef void (*GCThreadFunction)(void* param);
 // Interface that the GC uses to invoke OS specific functionality
 class GCToOSInterface
 {
+protected:
+    static size_t s_pageSize;
```

Comment:
@janvorli and @jkotas While I understand the concept, I do not understand how it would be implemented.  Which files would include the new inl files.  How would they determine which one to include?